src/auth/auth.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Controller, Get, Post, Request, UseGuards } from '@nestjs/common';
+import { LocalAuthGuard } from './local-auth.guard';
+import { AuthService } from './auth.service';
+import { JwtAuthGuard } from './jwt-auth.guard';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @UseGuards(LocalAuthGuard)
+  @Post('/login')
+  async login(@Request() req) {
```

Comment:
Should we add return type?
If you return entire ...req.user object, is there potentially too much data like  password or currentToken ?